.travis.yml

```diff
@@ -1,47 +1,37 @@
+dist: bionic
 language: python
 branches:
   only:
-  - master
+    - master
 cache:
-- pip
-- apt
+  - pip
+  - apt
 python:
-- '3.6'
+  - '3.6'
```

Comment:
we should add 3.7, should we?

Reply:
(could be a separate pr of course)

Reply:
@syphar yeah, I was adding it in the beginning, but decided to limit myself with just config redesign. Adding new version can go separately.